lib/main_common.pm

```diff
@@ -1894,6 +1900,10 @@ sub load_x11_message {
         loadtest "x11/evolution/evolution_timezone_setup";
         loadtest "x11/evolution/evolution_meeting_imap";
         loadtest "x11/evolution/evolution_meeting_pop";
+        if (!is_pre_15 && (!is_server() || we_is_applicable())) {
+            loadtest "x11/thunderbird/thunderbird_imap";
+            loadtest "x11/thunderbird/thunderbird_pop";
+        }
```

Comment:
hm, here you are adding the same test modules again? So the above is for openSUSE only? Please try to select an approach which works the same for o3 and osd. We should have no problem to define test suites where needed to have both instances inline.

Reply:
yep, exactly! Ok, if we add a qam-regression-message test suite also in o3 then we can simply remove this code.

Reply:
@okurz do I have to open a bug in poo or in boo ?